File: service/php-symfony/src/Controller/AddAccount/AccountRequest.php
<?php

namespace App\Controller\AddAccount;

use App\Entity\Account;

class AccountRequest
{
    public string $login;

    function toAccountEntity(): Account {
        $account = new Account();
        $account->setLogin($this->login);
        $account->setCreationDate(new \DateTime());

        return $account;
    }
}

File: service/jvm-quarkus/src/main/java/com/ecobenchmark/controllers/addaccount/AccountRequest.java
package com.ecobenchmark.controllers.addaccount;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.quarkus.runtime.annotations.RegisterForReflection;

import java.time.Instant;
import java.util.UUID;

@RegisterForReflection
public class AccountRequest {

    private UUID id;

    private String login;

    @JsonProperty("creation_date")
    private Instant creationDate;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public Instant getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Instant creationDate) {
        this.creationDate = creationDate;
    }
}
